deployment/aws/.env.uat

```diff
@@ -0,0 +1,13 @@
+TITILER_STACK_NAME="dynamic-tiler"
```

Comment:
Since the `NASA-IMPACT` repo is meant to be project-ignorant, I think we can remove `.env.uat` and maybe even `.env.dev`